master/internal/task/allocation.go

```diff
@@ -305,7 +305,8 @@ func (a *allocation) SetProxyAddress(_ context.Context, address string) error {
 	defer a.mu.Unlock()
 
 	if len(a.req.ProxyPorts) == 0 {
-		return ErrBehaviorUnsupported{Behavior: "proxying"}
+		a.syslog.Debug("No ports to proxy. Skipping proxy registration.")
```

Comment:
```suggestion
		a.syslog.Debug("skipping proxy registration, no ports")
```

Reply:
this or just not print a message is fine with me.

Reply:
Being that there were tests failing because of this, I would like to get this in quickly.  I can make the suggested change the next time I make an OSS change. Thanks.